examples/advanced_diffusion_training/train_dreambooth_lora_sdxl_advanced.py

```diff
@@ -740,6 +749,40 @@ def parse_args(input_args=None):
     return args
 
 
+# Taken from B-LoRA repo https://github.com/yardenfren1996/B-LoRA/blob/main/blora_utils.py
+BLOCKS = {
+    "content": ["unet.up_blocks.0.attentions.0"],
+    "style": ["unet.up_blocks.0.attentions.1"],
+}
```

Comment:
Should we make this configurable instead? 

`content_b_lora_blocks` and `style_b_lora_blocks`?

Reply:
I wonder if we should already make this a single `--lora_unet_blocks` (or smth similar) to allow for full flexability on the targeted blocks for folks who want to experiment (re: @asomoza comments too), but have the default be the 2 blora blocks?

Reply:
if making it configurable, you'll have to check the values and error out when they don't exist. it would be nice if that also shows the possible correct values to use.